common/src/main/java/com/commercetools/payment/utils/impl/PaymentConnectorHelperImpl.java

```diff
@@ -20,7 +20,6 @@
     public CompletionStage<HttpRequestResult> sendHttpGetRequest(String url) {
         HttpRequest request = HttpRequest.of(HttpMethod.GET, url);
 
-        // TODO: must be injected, instead of initialization!!!
```

Comment:
It's not planned to be injected anymore? If so, why?

Reply:
this actually was a wrong comment: it is not a sphere client, it is creating a new HttpClient to make an HTTP request (not to the platform)